swingset/src/main/java/com/nqadmin/swingset/models/SSListItemFormat.java

```diff
@@ -241,10 +241,10 @@ protected void appendValue(StringBuffer _sb, int _elemIndex, ListItem0 _listItem
 		case TIME:
 		case TIMESTAMP:
 			String pattern = patterns.get(jdbcType);
-			if (pattern != null) {
+			if (pattern != null && elem != null) {
 				SimpleDateFormat dateFormat = new SimpleDateFormat(pattern);
 				dateFormat.format(elem, _sb, FP0);
-			} else {
+			} else if(elem != null){
 				_sb.append(elem.toString());
 			}
```

Comment:
Definitely needs a fix. Taking a cue from the `default:` I might have wrapped this whole case in a single 
```
if(elem != null) {
    ...
}
```